neonvm/samples/vm-example.dimmslots.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       min: 1
       max: 2
       use: 2
-    memoryProvider: VirtioMem
+    memoryProvider: DIMMSlots
```

Comment:
Do we still need this sample file, keeping in mind that we have explicit error return `errors.New("DIMMSlots memory provider is deprecated and disabled")` for the DIMMSlots value?